### DISCUSSION TITLE
FATAL INPUT ERROR: unrecognized parameter name "--runMode" in input "Command-Line-Initial"

### DISCUSSION THREAD

**Discussion by SaswatiKar** [alexdobin/STAR#1882] [Q&A]
Hi,
I have downloaded and compiled STAR version 2.7.10b_alpha_23-06-09. I am getting the following error while trying to generate the genome index files.

EXITING: FATAL INPUT ERROR: unrecognized parameter name "--runMode" in input "Command-Line-Initial"
SOLUTION: use correct parameter name (check the manual)

Jun 17 04:40:07 ...... FATAL ERROR, exiting

I have my STAR directory as well as the reference genome fasta and gtf files in the same parent directory. This is the command I ran:

`$ ./STAR/source/STAR --runMode genomeGenerate --genomeDir ./Index_files --genomeFastaFiles ./Mus_musculus.GRCm39.dna.primary_assembly.fa --sjdbGTFfile ./Mus_musculus.GRCm39.109.gtf --sjdbOverhang 100`

I do not see any log.out file generated, neither in my output directory for index files (which remains empty) nor in the working directory.

Based on earlier discussions regarding similar cases of error for unrecognized parameter name in input 'command line initial', the error was during the alignment step and the issue appeared to be variations between STAR versions used for indexing and alignment, but this is the first instance of me using STAR in my system and for indexing.

Any help would be greatly appreciated!

Best,
Saswati

**Comment by alexdobin** [comment]
Hi Saswati,

not sure what causes this. I tested --runMode genomeGenerate and it works fine.
I would recommend trying a different version of STAR, or running it on a different computer.
